huaweicloud/services/dds/resource_huaweicloud_dds_instance_v3.go

```diff
@@ -821,6 +846,11 @@ func getDdsInstanceV3MongosNodeID(client *golangsdk.ServiceClient, d *schema.Res
 
 func flavorUpdate(ctx context.Context, config *config.Config, client *golangsdk.ServiceClient, d *schema.ResourceData,
 	opts []instances.UpdateOpt) error {
+	err := waitForInstanceReady(ctx, client, d.Id(), d.Timeout(schema.TimeoutUpdate))
```

Comment:
@Jason-Zhang9309 Can you add a specification for which parameters should wait for a ready state before updating?

Reply:
there is no description about this in the API doc
I add the waitForInstanceReady in where we call the update APIs, just in case